command/agent/config_test.go

```diff
@@ -84,6 +84,7 @@ func TestConfig_Merge(t *testing.T) {
 			},
 			NetworkSpeed:   100,
 			CpuCompute:     100,
+			MemoryMB:       200,
```

Comment:
where is this coming from? Same with line 231

Reply:
This checks merging two configuration data structure works. Here we check that the values (thaty are arbitrary) are overriden by the other values (those below at line 231 probably)

Reply:
:+1: